src/helpers/getDXContractAddresses.js

```diff
@@ -17,7 +18,7 @@ module.exports = (web3, artifacts) => async (ContractName, dev = USE_DEV_CONTRAC
     ? '@gnosis.pm/dx-contracts/networks-dev.json'
     : '@gnosis.pm/dx-contracts/networks.json'
 
-  const networksJSON = Object.assign(require(networksFile), gnoNetworksJSON)
+  const networksJSON = Object.assign(require(networksFile), gnoNetworksJSON, wethNetworksJSON)
```

Comment:
Now we get WETH from utils automatically. It's already merged in add-migrations branch

Reply:
Let it be as an address getter